src/test/java/HomeWork22.java

```diff
@@ -0,0 +1,21 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class HomeWork22 extends BaseTest {
+    @Test
+    public void renamePlaylist() {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        loginPage.logIn();
+        homePage.choosePlaylist();
+        homePage.conTextClick();
+        homePage.newName();
+        Assert.assertTrue(homePage.checkPlayList());
+    }
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._